### PR TITLE
Provided icons for PHP, Rust and Swift instrumentations

### PR DESCRIPTION
**Pull request by uday060**

### CODE REVIEW

**Comment by cartermp** [comment]
@uday060 looks like there's a build error here. If you run `npm run serve` locally it should reproduce on your end.

File: content/en/docs/instrumentation/php/_index.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 title: PHP
-description: A language-specific implementation of OpenTelemetry in PHP.
+description: >-
+  <img width="35" src="https://github.com/open-telemetry/opentelemetry.io/blob/main/iconography/32x32/PHP.svg"></img>
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/_index.md:4]: Update with URL to raw.github:

https://raw.githubusercontent.com/open-telemetry/opentelemetry.io/main/iconography/32x32/PHP.svg

File: content/en/docs/instrumentation/rust/_index.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 title: Rust
-description: A language-specific implementation of OpenTelemetry in Rust.
+description: >-
+  <img width="35" src="https://github.com/open-telemetry/opentelemetry.io/blob/main/iconography/32x32/Rust.svg"></img>
```
Comment (svrnm) [inline content/en/docs/instrumentation/rust/_index.md:4]: Update with URL to raw.github:

https://raw.githubusercontent.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Rust.svg

File: content/en/docs/instrumentation/swift/_index.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ---
 title: Swift
 weight: 28
-description: A language-specific implementation of OpenTelemetry in Swift.
+description: >-
+  <img width="35" src="https://github.com/open-telemetry/opentelemetry.io/blob/main/iconography/32x32/Swift.svg"></img>
```
Comment (svrnm) [inline content/en/docs/instrumentation/swift/_index.md:5]: Update with URL to raw.github:

https://raw.githubusercontent.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Swift.svg

**Review by svrnm — CHANGES_REQUESTED** [review]

File: content/en/docs/instrumentation/php/_index.md (hunk 1 above)
Comment (uday060) [reply content/en/docs/instrumentation/php/_index.md]: Done

File: content/en/docs/instrumentation/rust/_index.md (hunk 2 above)
Comment (uday060) [reply content/en/docs/instrumentation/rust/_index.md]: Done

File: content/en/docs/instrumentation/swift/_index.md (hunk 3 above)
Comment (uday060) [reply content/en/docs/instrumentation/swift/_index.md]: Done

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
@uday060, thanks for helping, looks good to me:)

**Review by cartermp — APPROVED** [review]

**Pull request merged**